cmake/Readme.md

```diff
@@ -0,0 +1,23 @@
+## The CMake for Mbed OS
+
+Until further notice, CMake for Mbed OS is still in the development.  
+
+### Targets
+
+Only limited set of targets is supported at the moment.
+
+These targets should work:
+- K64F 
+- DISCO_L475VG_IOT01A 
+- NRF52840_DK 
+- WIO_3G
+
+We will be adding more targets.
+
+### Examples
+
+Supported examples must contain CMakelists.txt. If it is not present, an example needs to be ported to support CMake.
+
+### Contributions are welcome!
+
```

Comment:
Contributions are accepted under the same license mentioned in the top level [`README.md`](..\README.md).